hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/totalsupply/TotalSupplyCall.java

```diff
@@ -46,6 +46,6 @@ public TotalSupplyCall(
     protected @NonNull FullResult resultOfViewingToken(@NonNull Token token) {
         final var output =
                 TotalSupplyTranslator.TOTAL_SUPPLY.getOutputs().encodeElements(BigInteger.valueOf(token.totalSupply()));
-        return successResult(output, 0L);
+        return successResult(output, 100L);
```

Comment:
Should be `gasCalculator.viewGasRequirement()` but this will be fixed in [this PR](https://github.com/hashgraph/hedera-services/pull/11148) so no worries here.